packages/styles/scss/patterns/sub-patterns/link-list/README.md

```diff
@@ -0,0 +1,23 @@
+### SCSS
+
+#### Usage
+
+Import the package css into the top of your main CSS file.
+
+```css
+@import '@carbon/ibmdotcom-styles/scss/patterns/sub-patterns/card/index';
+```
+
+#### Blocks
+
+| Class             | Description                                     |
+| ----------------- | ----------------------------------------------- |
+| `.bx--link--list` | Highest level class name for the Card component |
+
+#### Elements
+
+| Class                      | Description                                |
+| -------------------------- | ------------------------------------------ |
+| `.bxlink--list__heading`   | Heading element of the LinkList component. |
+| `.bxink--list__list`       | List of items for LinkList.                |
+| `.bxlink--list__list__CTA` | Within each item is leveraging the CTA.    |
```

Comment:
These class names seem a bit out of whack. Can these be fixed? Should the base classname be `.bx--link-list`?

Reply:
sorry, its my bad. will update